.circleci/config.yml

```diff
@@ -1273,7 +1273,7 @@ jobs:
       - run:
           name: Install runtime dependencies
           command: |
-            pacman --noconfirm -Syu --noprogressbar --needed base-devel boost cmake z3 cvc4 git openssh tar
+            pacman --noconfirm -Syu --noprogressbar --needed base-devel boost cmake z3 git openssh tar
```

Comment:
I don't actually understand why `z3` is installed for `t_archlinux_soltest` but not for `b_archlinux`.

Reply:
Yeah, I was about to comment about it. You can remove this step. It is already installed in `b_archlinux`.

Reply:
Yes, everything except `z3` 😄 

I am also wondering about this in ` b_archlinux`
```
# This can be switched off if we run out of sync with Arch.
      USE_Z3: ON
```

Should we switch it off?

Reply:
No, keep it for now, and also installing `z3` in that step. We actually need to think about if we will reenable the tests once we update the z3 version (i.e. removing the `--no-smt` flag). But for now let's keep that flag to disable the tests.

Reply:
Changed the step accordingly and the CI is still green.